client/v2/autocli/flag/address.go

```diff
@@ -147,20 +146,18 @@ func (a *consensusAddressValue) Set(s string) error {
 	return nil
 }
 
+// getKeyringFromCtx retrieves the keyring from the provided context.
+// If the context is nil or does not contain a valid client context,
```

Comment:
Shall we fallback and check `client.Context` v1 of we cannot find an autocli client context?
Just to be sure